Lib/dis.py

```diff
@@ -330,6 +330,84 @@ class Instruction(_Instruction):
                      covered by this instruction
     """
 
+    @staticmethod
+    def _get_argval_argrepr(op, arg, offset, co_consts, names, varname_from_oparg):
```

Comment:
`_get_instructions_bytes` is sufficiently long, about 120 lines, that pulling half of it out into a separate, coherent function seems sensible in itself.  Should it get a new, separate test?

Reply:
Good point, I've added a test.